mitodl/open-discussions
# import podcast data

#### Pre-Flight checklist

#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/2736

#### What's this PR do?
This pr ingests podcast data into the course catalog

#### How should this be manually tested?
Set `OPEN_PODCAST_DATA_BRANCH=ab/test-ingest`
Run
`docker-compose run web ./manage.py backpopulate_podcast_data`
Verify that podcasts and podcast episodes are imported and that the first podcast imported has topics and "Custom podcast title" as the title
